wordlist.py

```diff
@@ -354,6 +354,46 @@ class MissingTranslationException(Exception):
             'Desoladamente'
         ]
     ],
+	"nb-NO": [
```

Comment:
This should be nb_NO so it's like the other IETF tags. Also there's tabs instead of 4 spaces, would be great if you could replace those.